features/createroom/impl/src/main/kotlin/io/element/android/features/createroom/impl/configureroom/ConfigureRoomPresenter.kt

```diff
@@ -39,21 +52,47 @@ class ConfigureRoomPresenter @Inject constructor(
             }
         }
 
+        val localCoroutineScope = rememberCoroutineScope()
+        val createRoomAction: MutableState<Async<RoomId>> = remember { mutableStateOf(Async.Uninitialized) }
+
+        fun createRoom(config: CreateRoomConfig) {
+            createRoomAction.value = Async.Uninitialized
```

Comment:
Nit: setting to `Uninitialized` could be done in the fun `createRoom()`

Reply:
https://github.com/vector-im/element-x-android/pull/324/commits/f566a543e4913b84e6ea596e685ee1dafe2eeeea